src/utils/__tests__/deleteEmptyValuesFromObject.test.ts

```diff
@@ -0,0 +1,32 @@
+import deleteEmptyValuesFromObject from '../deleteEmptyValuesFromObject';
+
+test('deletes properties from object when they have empty value', () => {
+  const testValues = [
+    {
+      param: {},
+      expected: {},
+    },
+    {
+      param: { test1: '', test2: 'asd' },
+      expected: { test2: 'asd' },
+    },
+    {
+      param: {
+        test1: [],
+        test2: '',
+        test3: null,
+        test4: undefined,
+        test5: ['asd', 'osd'],
+        test6: 'moi',
+      },
+      expected: {
+        test5: ['asd', 'osd'],
+        test6: 'moi',
+      },
+    },
+  ];
+
+  testValues.forEach((t) => {
+    expect(deleteEmptyValuesFromObject(t.param)).toEqual(t.expected);
+  });
+});
```

Comment:
Needless if lodash (pickBy) is used instead of custom function.

Reply:
We could still abstract that one inside `deleteEmptyValuesFromObject` and leave the tests :)

Reply:
I think it is easier to understand `deleteEmptyValuesFromObject(object)` than `_.omitBy(object, (v) => _.isNil(v) || _.isEmpty(v))`

Reply:
hmm or `deleteEmptyPropertiesFromObject(object)`